roofit/roofitcore/inc/RooHistPdf.h

```diff
@@ -96,8 +96,11 @@ class RooHistPdf : public RooAbsPdf {
 
   void computeBatch(cudaStream_t*, double* output, size_t size, RooFit::Detail::DataMap const&) const override;
 
-protected:
+  void translate(RooFit::Detail::CodeSquashContext &ctx) const override;
+  std::string
+  buildCallToAnalyticIntegral(Int_t code, const char *rangeName, RooFit::Detail::CodeSquashContext &ctx) const override;
```

Comment:
```suggestion
  buildCallToAnalyticIntegral(int code, const char *rangeName, RooFit::Detail::CodeSquashContext &ctx) const override;
```
Let's avoid these old `Int_t` aliases.